### PR TITLE
Release 1.2.2

### PR DESCRIPTION
**Pull request by mikedep333**
I deleted the 1.2.1 release, but left the tag there.

The problem with it is that I forgot to bump the version string.

when I release 1.2.2, I'll do the changelog since 1.2.0.

### CODE REVIEW

**Comment by mikedep333** [comment]
Merge #29 1st.

File: pulpcore.te [hunk 1]
```diff
@@ -1,4 +1,4 @@
-policy_module(pulpcore, 1.2.0)
+policy_module(pulpcore, 1.2.2)
```
Comment (fao89) [inline pulpcore.te:1]: why are we moving from 1.2.0 to 1.2.2?

Comment (fao89) [reply pulpcore.te:1]: forget it, I read the description now

**Review by fao89 — APPROVED** [review]
Thank you!

**Review by dkliban — APPROVED** [review]

**Pull request merged**